platform/docs/docs/platform/modes/index.md

```diff
@@ -344,6 +346,7 @@ function modeFactory() {
     id: '',
     id: '',
     displayName: '',
+    hotkeyName: 'myHotkeyLocalStorageName',
```

Comment:
It is weird that we have `hotkeyName` and then below we have `hotkeys` hmmmmmm

Does it make sense to have the `hotkeys` array an array of objects 

```js
hotkeys: [{name: myHotKeyLocal, hotkeys:[{....}]}]
```

so that we use the first name be default, BUT if the mode wants it can run with the other one? and later we can have the UI to switch between them?

Reply:
That would be incompatible with what exists now, and we are starting to have modes.  Mostly, all that needs to be specified by the mode is the hotkeyName, and then it can get the default values.

Reply:
I think it actually might make sense at some point to define the set of hotkeys elsewhere entirely, and just be able to read them by hotkey name.

Reply:
Yeah I think we need hotkeyModule, but at least bring the hotkeyName close to the definition of hotkeys